src/algorithm/joint-configuration.hpp

```diff
@@ -251,6 +251,38 @@ namespace pinocchio
     dIntegrate<LieGroupMap,Scalar,Options,JointCollectionTpl,ConfigVectorType,TangentVectorType,JacobianMatrixType>(model, q.derived(), v.derived(), J.derived(),arg);
   }
 
+  /**
+   * @brief      Overall squared distance between two configuration vectors
+   *
+   * @param[in]  model      Model we want to compute the distance
+   * @param[in]  q0         Configuration 0 (size model.nq)
+   * @param[in]  q1         Configuration 1 (size model.nq)
+   *
+   * @return     The squared distance between the two configurations
+   */
+  template<typename LieGroup_t, typename Scalar, int Options, template<typename,int> class JointCollectionTpl, typename ConfigVectorIn1, typename ConfigVectorIn2>
+  Scalar squaredDistanceSum(const ModelTpl<Scalar,Options,JointCollectionTpl> & model,
```

Comment:
I don't this version is really worthy

Reply:
@gabrielebndn Do you have ideas of when this function might be useful?
Maybe @jmirabel has some hints.